src/Listeners/AddApiRoutes.php

```diff
@@ -38,8 +38,8 @@ public function routes(ConfigureApiRoutes $routes)
         /**
          * Forum side
          */
-        $routes->get('/masquerade/profile/{id:[0-9]+}', 'masquerade.api.profile', UserProfileController::class);
-        $routes->get('/masquerade/configure', 'masquerade.api.configure', UserConfigureController::class);
-        $routes->post('/masquerade/configure', 'masquerade.api.configure.save', UserConfigureController::class);
+        $routes->get('/u/{username}/profile/{id:[0-9]+}', 'masquerade.api.profile', UserProfileController::class);
+        $routes->get('/profile/configure', 'masquerade.api.configure', UserConfigureController::class);
+        $routes->post('/profile/configure', 'masquerade.api.configure.save', UserConfigureController::class);
```

Comment:
I don't think it's a good idea to rename the routes. `profile` is far too generic, it might be used by another extension.

I would even be in favour of using `/flarum/masquerade` prefix for all api calls, like I did in Mason https://github.com/flagrow/mason/blob/master/src/Listeners/AddApiRoutes.php